src/test/resources/unit/rest-project/pom.xml

```diff
@@ -7,18 +7,18 @@
 
   <groupId>com.github.cjnygard</groupId>
   <artifactId>rest-maven-plugin-test</artifactId>
-  <version>0.1.0-SNAPSHOT</version>
+  <version>0.1.5-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <name>REST Request Maven Plugin Test</name>
 
   <properties>
-    <rest-maven-plugin.version>0.1.0-SNAPSHOT</rest-maven-plugin.version>
+    <rest-maven-plugin.version>0.1.5-SNAPSHOT</rest-maven-plugin.version>
```

Comment:
Can you change 0.1.5-SNAPSHOT to ${version}   then I'll squash-merge the patches.